<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbChannelControl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Channel Control</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Only move to channels under the parent selected below: -</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeParentChannel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="allColumnsShowFocus">
     <bool>true</bool>
    </property>
    <property name="headerHidden">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">Room</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Channel</string>
     </property>
    </column>
   </widget>
   <widget class="QRadioButton" name="rbExpertMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Automatic - move on retune - no manual channel changes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbDisabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Disabled - Don't automove based on simulator tuning at all</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbEasyMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Semi-automatic - move on retune - manual channel changes too</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbConsiderRange">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Do not move to a station that is &quot;out of range&quot;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbManualModeOnStart">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>321</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Revert to allowing manual channel changes on initial load</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbUntuned">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>'Untuned' Behaviour</string>
   </property>
   <widget class="QRadioButton" name="rbUntunedStay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Stay in current channel if COM tuning doesn't match TS channels</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbUntunedMove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>351</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Move to 'untuned' channel if COM tuning doesn't match TS channels</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeUntunedChannel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="allColumnsShowFocus">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">Room</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">Channel</string>
     </property>
    </column>
   </widget>
   <widget class="QCheckBox" name="cbUntunedRange">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>61</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>'Out of Range' treated as 'Untuned'</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Use the selected channel when none matches the tuned frequency.</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Information Pane</string>
   </property>
   <widget class="QCheckBox" name="cbInfoDetailed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Detailed data display</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Range Effects</string>
   </property>
   <widget class="QCheckBox" name="cbRangeEffects">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Reduce station volume with range</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPlayStatic">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Static in background</string>
    </property>
   </widget>
  </widget>
  <zorder>gbUntuned</zorder>
  <zorder>buttonBox</zorder>
  <zorder>gbChannelControl</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
 </widget>
 <tabstops>
  <tabstop>rbDisabled</tabstop>
  <tabstop>rbEasyMode</tabstop>
  <tabstop>rbExpertMode</tabstop>
  <tabstop>cbConsiderRange</tabstop>
  <tabstop>treeParentChannel</tabstop>
  <tabstop>rbUntunedStay</tabstop>
  <tabstop>rbUntunedMove</tabstop>
  <tabstop>cbUntunedRange</tabstop>
  <tabstop>treeUntunedChannel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeParentChannel</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>Config</receiver>
   <slot>newRoot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeUntunedChannel</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>Config</receiver>
   <slot>newUntuned()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeUntunedChannel</sender>
   <signal>itemExpanded(QTreeWidgetItem*)</signal>
   <receiver>Config</receiver>
   <slot>columnResize(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeUntunedChannel</sender>
   <signal>itemCollapsed(QTreeWidgetItem*)</signal>
   <receiver>Config</receiver>
   <slot>columnResize(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeParentChannel</sender>
   <signal>itemCollapsed(QTreeWidgetItem*)</signal>
   <receiver>Config</receiver>
   <slot>columnResize(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeParentChannel</sender>
   <signal>itemExpanded(QTreeWidgetItem*)</signal>
   <receiver>Config</receiver>
   <slot>columnResize(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbDisabled</sender>
   <signal>clicked()</signal>
   <receiver>Config</receiver>
   <slot>modeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbEasyMode</sender>
   <signal>clicked()</signal>
   <receiver>Config</receiver>
   <slot>modeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbExpertMode</sender>
   <signal>clicked()</signal>
   <receiver>Config</receiver>
   <slot>modeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbUntunedStay</sender>
   <signal>clicked()</signal>
   <receiver>Config</receiver>
   <slot>untunedChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbUntunedMove</sender>
   <signal>clicked()</signal>
   <receiver>Config</receiver>
   <slot>untunedChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbConsiderRange</sender>
   <signal>toggled(bool)</signal>
   <receiver>Config</receiver>
   <slot>untunedChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newRoot()</slot>
  <slot>newUntuned()</slot>
  <slot>columnResize(QTreeWidgetItem*)</slot>
  <slot>modeChanged()</slot>
  <slot>untunedChanged()</slot>
 </slots>
</ui>
